src/ToDo/Business/UserProfilePictureService.cs

```diff
@@ -5,23 +5,20 @@ public class UserProfilePictureService : IUserProfilePictureService
 	private readonly IUserProfilePictureEndpoint _client;
 
 	private byte[]? _profilePicture;
-	private string? _userEmailAddress;
 
 	public UserProfilePictureService(IUserProfilePictureEndpoint client)
 	{
 		_client = client;
 	}
 
-	public async ValueTask<byte[]> GetAsync(string userEmailAddress, CancellationToken ct)
+	public async ValueTask<byte[]> GetAsync(CancellationToken ct)
```

Comment:
@agaldanaw2 why was this change made? The emailAddress was there to make sure the cache is flushed if the signed in user changes?

Reply:
cc @carldebilly @agneszitte-nventive

Reply:
Resolving the comment as you already made the proper changes to fix it